sw/host/opentitanlib/src/transport/ti50emulator/mod.rs

```diff
@@ -74,6 +75,7 @@ impl Ti50Emulator {
             instance_directory,
             process: RefCell::new(process),
             gpio_map: RefCell::new(HashMap::new()),
+            uarts: RefCell::new(Vec::new()),
```

Comment:
IIRC, the Default impl for containers is an empty container, and I think `RefCell<T>` implements `Default` if `T: Default`.

Depending on your sense of style, you could write `RefCell::default()` here. (I'm find if you want to leave it as-is also).

Reply:
This is good to know.  Done.